torch/csrc/api/include/torch/nn/functional/linear.h

```diff
@@ -6,6 +6,11 @@ namespace torch {
 namespace nn {
 namespace functional {
 
+inline Tensor bias(const Tensor& input, const Tensor& bias){
+  // Do I need to check the constraints here?
+  return torch::bias(input, bias);
```

Comment:
by using the torch:: namespace am I calling the native function defined in linear.cpp

For both python and c++ api the calling structure should be? Module -> functional -> native ?